### PR TITLE
Move testimonials to custom model

### PR DESCRIPTION
**Pull request by ezwang**
- Move testimonials to a custom model to make it more flexible to style on the frontend.
- Add management script to convert existing testimonials in descriptions to new format.

@ArmaanT  will need a database backup before running this script. Will run script after this PR is merged!

### CODE REVIEW

File: clubs/management/commands/extract_testimonials.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from clubs.utils import clean
+
+
+class Command(BaseCommand):
+    help = 'Extract club testimonials from descriptions and remove testimonials from descriptions.'
+
+    def handle(self, *args, **kwargs):
+        for club in Club.objects.all():
+            if club.description:
+                match = re.match(r'(.*)From members of ([^:]+)[:;](.*)', club.description, re.M | re.I | re.S)
```
Comment (cacabo) [inline clubs/management/commands/extract_testimonials.py]: Is this like a common term we have in club descriptions? If so might be worth putting in a comment explaining that that's the case or giving an example—I always find regexps hard to parse from a readability POV

Comment (ezwang) [reply clubs/management/commands/extract_testimonials.py]: This is the format that @valenciafu imported all of the testimonials in, and it's actually super consistent except for one out of ~90 cases where someone accidentally used a semicolon instead of a colon.

I expect to run this script exactly once and then never again, but I'll document the regex for future purposes!

**Comment by ezwang** [comment]
> Also, is there a clubhouse story for updating the frontend to render testimonials info?

The frontend is finished in the `fair` branch on the other repository, just need @valenciafu to style it and it should be ready for deployment.
